packages/keyring-eth-hd/src/hd-keyring.ts

```diff
@@ -567,8 +561,8 @@ export class HdKeyring {
       assert(privateKey, 'Expected private key to be set');
       const appKeyOriginBuffer = Buffer.from(withAppKeyOrigin, 'utf8');
       const appKeyBuffer = Buffer.concat([privateKey, appKeyOriginBuffer]);
-      const appKeyPrivateKey = arrToBufArr(keccak256(appKeyBuffer));
-      const appKeyPublicKey = privateToPublic(appKeyPrivateKey);
+      const appKeyPrivateKey = Buffer.from(keccak256(appKeyBuffer));
```

Comment:
Is it expected to still use `Buffer` instead of `Uint8Array`?

Reply:
I mentioned in the PR description that I wanted to do the minimum changes for the sake of stability, so some of the private and helper methods still rely on `Buffer` which I haven't changed.

It should be harmless since `Buffer` is a subset of `Uint8Array` meaning every `Buffer` is an `Uint8Array`, where as every `Uint8Array` isn't a `Buffer`.